extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -305,6 +312,14 @@ public void setSecretProvider(Provider secretProvider) {
                 this.secretProvider = secretProvider;
             }
 
+            public Optional<String> getSignatureAlgorithm() {
+                return signatureAlgorithm;
+            }
+
+            public void setSignatureAlgorithm(String signatureAlgorithm) {
+                this.signatureAlgorithm = Optional.of(signatureAlgorithm);
+            }
+
```

Comment:
Why the getters and setters? The other config classes don't have this

Reply:
@gastaldi Sure, this convention is followed in OIDC and OIDC client because the configuration can be created manually, example, with `quarkus-oidc` `TenantConfigResolver` and `quarkus-oidc-client` `OidcClients` - so we have setters as well for the user code not to do `OidcClientConfig.a.b` or `OidcTenantConfig.a.b` but have `setA` etc.

Awhile back we talked with Pedro about possibly providing a configuration builder for OIDC - but there are so many properties and groups it would be hard to maintain :-)